### DISCUSSION TITLE
v3 is around the corner. Is it worth starting a custom theme in v2?

### DISCUSSION THREAD

**Discussion by pekka** [filamentphp/filament#7008] [General]
Building my first app with Filament (and Laravel for that matter) and stoked about the wealth of convenient solutions to almost every problem, the fantastic documentation, and the good vibes Filament's creators and community are giving off ❤️ thinking this could be the beginning of a long friendship....

The app I am building will be fairly complex, and I want to change Filament's UI a little bit towards a horizontal layout with dropdown menus, not unlike Tabler does it:

![image](https://github.com/filamentphp/filament/assets/262464/fffe7127-8e4c-4b38-aada-aa8f0b8e7ca1)

However, Filament v3 is around the corner and I am wondering whether it is worth delving into how to best build a Filament backend theme in a v2 context. Or is it better to wait for v3 because there will be breaking changes to how theming is handled.

**Comment by danharrin** [comment]
Filament v3 has the option of a horizontal navigation instead of a sidebar, so you won't need a theme for it
